prowler/lib/scan/scan.py

```diff
@@ -18,6 +19,7 @@ class Scan:
     _service_checks_completed: dict[str, set[str]]
     _progress: float = 0.0
     _findings: list = []
+    _duration: float = 0.0
```

Comment:
I think an `int` is more than enough here.

Reply:
Done